pkgs/development/interpreters/python/hooks/default.nix

```
@@ -107,7 +106,7 @@ in rec {
       };
     } ./python-imports-check-hook.sh) {};
 
-  pythonNamespacesHook = callPackage ({}:
+  pythonNamespacesHook = callPackage ({ findutils }:
     makeSetupHook {
       name = "python-namespaces-hook.sh";
```

```suggestion
      name = "python-namespaces-hook.sh";
      deps = [ findutils ];
```
This is how all the other hooks are doing it.

---

not necessary, please undo this change. findutils is already substituted below